
---
bool answer_received
int16 asa_mx
int16 asa_my
int16 asa_mz
